exasol/analytics/audit/columns.py

```diff
@@ -0,0 +1,61 @@
+from exasol.analytics.schema import (
+    decimal_column,
+    timestamp_column,
+    varchar_column,
+)
+
+
+class AuditColumns:
```

Comment:
Could maybe an enum

Reply:
```suggestion
class BaseAuditColumns:
```

Reply:
An Enum requires to use `AuditColumns.ROWS_COUNT.value` when accessing an entry of the enum. 

In order to avoid this, I chose not to use an enum.